sdk/core/src/main/java/com/klaviyo/core/lifecycle/KlaviyoLifecycleMonitor.kt

```diff
@@ -70,5 +84,13 @@ internal object KlaviyoLifecycleMonitor : LifecycleMonitor, Application.Activity
         // Warning: onActivityDestroyed is unreliable, I'm not even going to try to broadcast it
     }
 
+    override fun onConfigurationChanged(newConfig: Configuration) {
```

Comment:
nit: Not technically an activity lifecycle callback (right?), should we move this below the `//endregion` comment that denotes our block for activity lifecycle callbacks in this object?